lib/common/error_private.h

```diff
@@ -51,7 +51,8 @@ typedef ZSTD_ErrorCode ERR_enum;
 #ifdef ERROR
 #  undef ERROR   /* reported already defined on VS 2015 (Rich Geldreich) */
```

Comment:
very minor : 
it's allowed to `#undef` an identifier without previous definition.
So the `#ifdef` guard is not useful.